mautic/mautic
# [UI] Change Form, SMS and Email types header

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

We recently created a pull request to improve the comparison of different forms and the community brought up the need to change other parts of the interface (see discussion here: https://github.com/mautic/mautic/pull/13381#issuecomment-1954413474).

Specifically they asked to reduce the size of the title of each form to remove the word "New".

Additionally, I also changed the icons for the form types and email types using Remix icon.

![image](https://github.com/mautic/mautic/assets/116097999/e8c91385-c5b9-412d-8b9d-7c797a4e77dc)

![image](https://github.com/mautic/mautic/assets/116097999/16dd0553-e3e7-44f7-a6c5-17928b628647)

![image](https://github.com/mautic/mautic/assets/116097999/ddf43beb-702a-4b98-b20b-f681c841d096)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Components > Forms > Create new > see new labels and icons
3. Open Channels > Emails > Create new > see new labels and icons
4. Open Plugins > Twilio > set Published to Yes and fill Account SID with "abc" > Open Channels > Text Messages > Create new > see new labels and icons
